### PR TITLE
Update docs.md: use `make.bat` under Windows.

### PR DESCRIPTION
**Pull request by changrui**

### CODE REVIEW

File: doc/docs.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ You need `git`, and a C compiler like `gcc` or `msvc`:
 ```bash
 git clone https://github.com/vlang/v
 cd v
-make
+make.bat -gcc
```
Comment (spytheman) [inline doc/docs.md:33]: Good spot.
`make.bat -tcc` will be better for most people imho.

Comment (spytheman) [reply doc/docs.md:33]: The reason is that we have a prebuilt tcc that the .bat file will simply download, and it is smaller than the gcc binary packages from say http://winlibs.com/

Comment (spytheman) [reply doc/docs.md:33]: Or just `make.bat` - I think in this case `make.bat` will by default use tcc also.

**Review by changrui — COMMENTED** [review]
Add `tcc`, `gcc`, `clang`, and `msvc` to compile `vlang`.

File: doc/docs.md [hunk 2]
```diff
@@ -26,11 +26,11 @@ git clone https://github.com/vlang/v && cd v && make
 ```
 
 ### Windows:
-You need `git`, and a C compiler like `gcc` or `msvc`:
+You need `git`, and a C compiler like `tcc`, `gcc`, `clang` or `msvc`:
 ```bash
-git clone https://github.com/vlang/v
+git clone -depth=1 https://github.com/vlang/v
```
Comment (spytheman) [inline doc/docs.md:31]: no need for that

File: doc/docs.md [hunk 3]
```diff
@@ -26,11 +26,11 @@ git clone https://github.com/vlang/v && cd v && make
...
 cd v
-make
+make.bat -tcc/-gcc/-msvc/-clang
```
Comment (spytheman) [inline doc/docs.md:33]: that makes the instructions harder to follow

**Pull request merged**